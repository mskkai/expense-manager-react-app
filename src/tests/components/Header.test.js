import React from "react";
import Header from "../../components/Header";
import { shallow } from "enzyme";

test("should render Header correctly using react test renderer", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
