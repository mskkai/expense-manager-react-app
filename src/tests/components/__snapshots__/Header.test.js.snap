// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly using react test renderer 1`] = `
<header>
  <h1>
     Extensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Go Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
